templates/cosmos-db.json

```diff
@@ -20,19 +27,25 @@
         "BoundedStaleness",
         "Strong",
         "ConsistentPrefix"
-      ]
-    },
-    "enableVirtualNetworkFilter": {
-      "type": "bool",
-      "defaultValue": false
+      ],
+      "metadata": {
+        "description": "The default consistency level and configuration settings of the Cosmos DB account"
+      }
     },
     "subnetResourceIdList": {
       "type": "array",
-      "defaultValue": []
+      "defaultValue": [
+      ],
+      "metadata": {
+        "description": ""
+      }
     },
     "ipRangeFilter": {
       "type": "string",
-      "defaultValue": ""
+      "defaultValue": "",
+      "metadata": {
+        "description": "Allowed list of client IPs for a given database account"
+      }
     }
   },
   "variables": {
```

Comment:
Given the complexity of the variables object I wonder if it's worth separating out these templates?

Reply:
I thought that it would be easy to add future additional account types to the template - EnableCassandra; EnableTable if required.  We can do one template for each if that is best.

Reply:
Taking advice.